app/src/components/listEntries/index.js

```diff
@@ -134,6 +134,10 @@ class ListEntries extends Component {
         <div className="clearfix dashboard-header-wrapper">
           <div className="left-content bulletin-heading">
             <h3>Bulletins</h3>
+            <strong>
+              Note: The Bulletins will take around {process.env.REACT_APP_BULLETIN_FETCH_INTERVAL} minutes to be Updated
```

Comment:
```suggestion
              Note: The Bulletins will take around {process.env.REACT_APP_BULLETIN_FETCH_INTERVAL} minutes to be Updated
              Note: The Bulletins will be refreshed every {process.env.REACT_APP_BULLETIN_FETCH_INTERVAL} minutes.
```
Removing "take around" and "automatically" part

Reply:
ok dai.